ber_test.go

```diff
@@ -44,7 +44,8 @@ func TestBer2Der_Negatives(t *testing.T) {
 		Input         []byte
 		ErrorContains string
 	}{
-		{[]byte{0x30, 0x85}, "tag length too long"},
+		{[]byte{0x30, 0x85}, "ber2der: cannot move offset forward, end of ber data reached"},
+		{[]byte{0x30, 0x85, 0x00}, "tag length too long"},
```

Comment:
I haven't taken the time to dig into the ber format and grok the changes above.  However, I'd say as long as we have test cases that caused the panic before, but no longer do, I'm happy.  Is `[]byte{0x30, 0x85, 0x00}` sufficient for this, or should we include other inputs?

Reply:
Short answer, no.  Though this was considered some.

Right now I have that coverage in Teleport's fuzzing: https://github.com/gravitational/teleport/pull/29231/files#diff-9356f3c834eb7d4e79f4bce10f28cb7f959930c03fcbe02cecd44323173dd676

I considered moving the fuzzing coverage to this repo, but I feel like `Teleport` is a better place for it due to how we have it setup with oss-fuzz, as well as being consistent with how we are testing other libraries.

I can add additional test cases for each of the corrected panic's rather than deferring to Teleport's testing.  That is probably a good recommendation.